lib/acts_as_paranoid/core.rb

```diff
@@ -124,6 +129,7 @@ def destroy!
             # Handle composite keys, otherwise we would just use `self.class.primary_key.to_sym => self.id`.
             self.class.delete_all(Hash[[Array(self.class.primary_key), Array(self.id)].transpose]) if persisted?
             self.paranoid_value = self.class.delete_now_value
+            self.updated_at = Time.now if self.class.paranoid_configuration[:touch]
```

Comment:
Do you think it makes sense to change the name of the option to avoid confusion?